lambdabot-core/result

```diff
@@ -0,0 +1 @@
+/nix/store/crakn4bicg213g3g429rk5zbr1q0z3z3-lambdabot-core-5.1.0.4
```

Comment:
Should these result files be included in the repository?

Maybe add a .gitingore rule to exclude them?